app/styles/tasks.css

```diff
@@ -36,49 +41,114 @@
 
 /* progress bar */
 
+.wrapper-div{
+  margin: 1em;
+  padding: 1rem;
+}
+
 .progress-bar {
-  background-color: #BFADA3;
-  height: 15px;
-  border-radius: 20px;
+  background-color: #fff;
+  height: 40px;
+  border-radius: 25px;
   overflow: hidden;
-  box-shadow: 2px 0 10px inset rgba(0,0,0,0.2);
   margin-top: 16px;
   position: relative;
 }
 
-.percent-20 {
-  width: 20%;
+.progress-bar-container{
+  padding: 1em;
 }
 
-.percent-40 {
-  width: 40%;
+input[type=range] {
+  -webkit-appearance: none;
+  width: 80%;
 }
 
-.percent-60 {
-  width: 60%;
+.input{
+  overflow: hidden;
 }
 
-.percent-80 {
-  width: 80%;
+input[type=range]:focus {
+  outline: none;
```

Comment:
I hope this won't be causing any issue for web accessibility and is tested however
Can we go for these options instead https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

Reply:
Have removed it, thanks for pointing out